src/__tests__/incidents/view/ViewIncidentDetails.test.tsx

```diff
@@ -41,160 +40,85 @@ describe('View Incident Details', () => {
 
     const history = createMemoryHistory()
     history.push(`/incidents/1234`)
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
-        <ButtonBarProvider.ButtonBarProvider>
-          <Router history={history}>
-            <ViewIncidentDetails incidentId="1234" permissions={permissions} />
-          </Router>
-        </ButtonBarProvider.ButtonBarProvider>,
-      )
-    })
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
-  }
-
-  describe('view details', () => {
-    it('should call find incident by id', async () => {
```

Comment:
Same question as https://github.com/HospitalRun/hospitalrun-frontend/pull/2516#discussion_r554037440 relating to a few tests in here